app/tests/route/test_resources.py

```diff
@@ -0,0 +1,220 @@
+                "conventional_count": 45,
+                "continue_care_count": 45,
+                "reanimation_count": 45,
+                "post_urgency_count": 54,
+                "other_count": 50
+            },
+            "supply": {
+                "respirators_count": 123,
+                "efp2_mask_count": 343,
+                "chir_mask_count": 345,
+                "blouses_count": 23,
+                "gowns_count": 12
+            },
+            "human": None
+        }
+
+        token = self.authenticate('joe@example.fr', 'super-secret-password')
+        url = 'api/resources'
+        response = self.post(url, token, params=params)
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response.json.get("id"), '1')
+        obj = Availability.query.get(1)
+        self.assertEqual(obj.json["date"], "2020-04-23T20:00:00")
+        self.assertEqual(obj.json["platform"],
+                         {'created_at': '2020-04-15T12:40:14.462544', 'updated_at': '2020-04-15T12:40:14.462544',
+                          'id': 1000, 'name': 'Covidmoiunlit'})
+        self.assertEqual(obj.json["organization"],
+                         {'address': {'created_at': '2020-04-15T12:40:14.462544',
+                                      'updated_at': '2020-04-15T12:40:14.462544', 'street': '30 rue des visiteurs',
+                                      'insee_code': None, 'zipcode': '54344', 'city': 'covid-19', 'lon': None,
+                                      'lat': None}, 'data': {'created_at': '2020-04-15T12:40:14.462544',
+                                                             'updated_at': '2020-04-15T12:40:14.462544', 'id': 1000,
+                                                             'finess_et': '12345567890', 'finess_ej': '12345567890'},
+                          'created_at': '2020-04-15T12:40:14.462544', 'updated_at': '2020-04-15T12:40:14.462544',
+                          'id': 1000, 'name': 'Hopital du Paradis', 'type': 1, 'reg_code': None, 'address_id': 1000})
+        self.assertEqual(obj.json["bed"],
+                         {'continue_care_count': 45, 'covid_used': 50, 'reanimation_count': 45, 'other_used': 25,
+                          'conventional_count': 45, 'other_count': 50, 'other_available': 150, 'covid_available': 100,
+                          'post_urgency_count': 54})
+        self.assertEqual(obj.json["supply"],
+                         {'blouses_count': 23, 'gowns_count': 12,
+                          'chir_mask_count': 345, 'efp2_mask_count': 343, 'respirators_count': 123})
+        self.assertIsNone(obj.json["human"])
+        for k, v in {'id': 1, "lastname": "Mistigri",
+                     "firstname": "Le Chat",
+                     "email": "mistigry.lechat@email.fr",
+                     "phone_number": "0123456789",
+                     "comment": "ne pas deranger ....."}.items():
+            self.assertEqual(obj.json["contact"][k], v)
```

Comment:
Couldn't we just validate the serialization of obj and the params?
```suggestion
        self.assertDictEqual(obj.json, params)
```

Reply:
I know it's not beautiful. But, it's the only mean I found to make test predictable. 
Cannot be done  in a other way depending on me, because there are the created_timestamp, which cannot be predictable.